File: spring-data/mongodb-api/src/main/java/com/github/mehdishahdoost/mongodbapi/service/StudentService.java
package com.github.mehdishahdoost.mongodbapi.service;

import com.github.mehdishahdoost.mongodbapi.model.Student;
import com.github.mehdishahdoost.mongodbapi.repository.StudentRepository;
import lombok.AllArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@AllArgsConstructor
public class StudentService {

    private StudentRepository  studentRepository;

    public List<Student> getStudents() {
        return studentRepository.findAll();
    }
}

File: mongodb-api-kotlin/src/main/kotlin/com/github/mehdishahdoost/service/StudentService.kt
package com.github.mehdishahdoost.service

import com.github.mehdishahdoost.model.Student
import com.github.mehdishahdoost.repository.StudentRepository
import lombok.AllArgsConstructor
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class StudentService @Autowired constructor(
   private val studentRepository: StudentRepository
) {
   fun getStudent(): List<Student> = studentRepository.findAll();
}